ci-operator/step-registry/optional-operators/README.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshooting olm deployment failures
+
+This document should help you to analyze where to look in case that olm deployment has failed. It also describes workflows and which one should be your main concern. Logs location is also mentioned as one of the main subjects.
+* [Workflows](#Workflows):
+* [Logs and how to get them](#Logs-and-how-to-get-to-them):
+
+## Workflows:
+You can find all the workflows that are being executed in following yaml files:
+https://github.com/openshift/release/tree/master/ci-operator/config/redhat-openshift-ecosystem/cvp
+
+I will use `4.7` ocp version in this example:
+The workflow you are most interested in is : **optional-operators-cvp-common-aws**.
+
+*Env variables* together with *dependenciy_overrides* are specified before the workflow, these are used during the steps which are mentioned in workflow itself.
+`CVP-COMMON-AWS` workflow does following: installs an optional operator using the input index image, package and channel and executes a common suite of CVP tests to validate the optional operator.
+
+In case you want to know what individual step does, search for them in https://github.com/openshift/release/tree/master/ci-operator/step-registry/optional-operators
+
+
+## Logs and how to get to them:
+If there is a problem with olm deployment in cvp product jenkins pipeline, specifically in *bundle-image-validation* job, such issue will be recorded to corresponding prow job.
+Which link is gonna look like this: 
+https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.7-cvp-common-aws/"unique-number".
+
+In this prow job you can inspect what exactly went wrong, which part of the workflow failed, etc.
+
+I will describe investigation of the issue in few steps:
```

Comment:
Generally in a how-to guide you want to avoid the first-person voice.  Try something like this instead:

`The steps to investigate the issue are as follow:`